CHANGELOG.md

```diff
@@ -5,7 +5,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [UNRELEASED][unreleased]
 
-## [1.6.0][1.6.0]
```

Comment:
You need to create a new heading and new links in CHANGELOG.

Reply:
Changed this and added new heading